server.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 // smtp network server
 
+// var log = require('why-is-node-running');
```

Comment:
BTW I left this in but commented because it makes it easy to debug any kind of "won't quit" issues in the future.